vql/tools/gcs_pubsub_publish.go

```diff
@@ -23,6 +23,7 @@ type GCSPubsubPublishArgs struct {
 	ProjectId   string      `vfilter:"required,field=project_id,doc=The project id to publish to"`
 	Msg         vfilter.Any `vfilter:"required,field=msg,doc=Message to publish to Pubsub"`
 	Credentials string      `vfilter:"required,field=credentials,doc=The credentials to use"`
+	Attributes  string      `vfilter:"required,field=attributes,doc=The publish attributes"`
```

Comment:
It would be nicer to have these as a dict so users can just easily give them:

```sql
SELECT gcs_pubsub_publish(...., attributes=dict(Foo=1)....)
```

Just specify the arg type as a `*ordereddict.Dict` and then you can just call its ToDict() method to get a map[]interface{}

Reply:
I'll try :)